util/util.go

```diff
@@ -384,13 +384,16 @@ var (
 
 // fileExistsAndNotADir - Check to see if a file exists
 // and that it is not a directory.
-func fileExistsAndNotADir(path string) bool {
+func fileExistsAndNotADir(path string) (bool, error) {
 	file, err := os.Stat(path)
 
-	if file == nil || err != nil || errors.Is(err, os.ErrNotExist) {
-		return false
+	if file == nil || err != nil {
+		if os.IsNotExist(err) {
```

Comment:
This should use errors.Is()! https://github.com/containers/buildah/pull/4143

Reply:
Why?  We are checking a returned status from os.Stat()  It should not be a wrapped error.

Reply:
> IsNotExist returns a boolean indicating whether the error is known to report that a file or directory does not exist. It is satisfied by ErrNotExist as well as some syscall errors.

> This function predates errors.Is. It only supports errors returned by the os package. New code should use errors.Is(err, fs.ErrNotExist). 

https://pkg.go.dev/os#IsNotExist

The point of that PR was to switch everything to errors.Is, you just change it back for no reason.

Reply:
Yes I went and read the new comment, and changed it back.